src/applications/personalization/rated-disabilities/components/RatedDisabilityList.jsx

```diff
@@ -45,10 +46,12 @@ class RatedDisabilityList extends React.Component {
         </>
       );
     } else {
-      headline = 'We don’t have rated disabilities on file for you';
       status = 'info';
       content = (
         <>
+          <h2 className="vads-u-margin-y--0 vads-u-font-size--lg">
```

Comment:
just curious if `vads-u-font-size--h3` would work here instead of `vads-u-font-size--lg` (not that it's necessarily better)